#!/usr/bin/env python
PACKAGE = "cmt"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_kp", double_t, 0, "Kp for x axis", 0.0, -2, 2)
gen.add("x_kd", double_t, 0, "Kd for x axis", 0.0, -2, 2)

gen.add("y_kp", double_t, 0, "Kp for y axis", 0.0, -2, 2)
gen.add("y_kd", double_t, 0, "Kd for y axis", 0.0, -2, 2)

gen.add("z_kp", double_t, 0, "Kp for z axis", 0.0, -2, 2)
gen.add("z_kd", double_t, 0, "Kd for z axis", 0.0, -2, 2)

exit(gen.generate(PACKAGE, "cmt", "tuning"))
